### PR TITLE
Expose gmt bedpe on the command line

### PR DESCRIPTION
**Pull request by apregier**

### CODE REVIEW

File: lib/perl/Genome/Model/Tools/Bedpe.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package Genome::Model::Tools::Bedpe;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Bedpe {
+    is => ['Command'],
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Bedpe.pm]: Should this be a `Command::Tree`?

Comment (apregier) [inline lib/perl/Genome/Model/Tools/Bedpe.pm]: Maybe?  Seems to work as is though:
> gmt bedpe -h
Using libraries at /gscuser/aregier/git/genome/copy/lib/perl
Sub-commands for gmt bedpe:
 evaluate-bedpe  --bedpe=? --gold-...  no description!!!: define 'doc' in the class definition for
                                        Genome::Model::Tools::Bedpe::EvaluateBedpe

Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Bedpe.pm]: Aye, `Command` is `Command::V1`.  For `Command::V2`, the concept of "parent of subcommands" was split out into `Command::Tree`.  (In `V1` we could often have something be both the base class for a tree of commands and the parent of it, but now they must be distinct.)

File: lib/perl/Genome/Model/Tools/Bedpe.pm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+};
+
+sub help_brief {
+    "Tools related to BEDPE files.",
+}
+
+sub help_synopsis {
+    my $self = shift;
```
Comment (dufeiyu) [inline lib/perl/Genome/Model/Tools/Bedpe.pm]: unneeded $self

**Comment by apregier** [comment]
Pushed commits to address the comments, thanks!

**Comment by dufeiyu** [comment]
+1

**Comment by tmooney** [comment]
+1 :chestnut:

**Pull request merged**